example_sns_init.yaml

```diff
@@ -61,7 +106,7 @@ NnsProposal:
 
 # If the SNS launch attempt fails, control over the dapp canister(s) is given to
 # these principals. In most use cases, this is chosen to be the original set of
-# controller(s) of the dapp.
+# controller(s) of the dapp. Must be a list of PrincipalIds.
```

Comment:
We currently do not refer to the Rust types but to the IC concepts in plane English.

Let's therefore say `Must be a list of plincipals.`